dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -271,6 +268,14 @@ KRB5_FILE_FOUND=CWIML4516E: The LdapRegistry component is configured to use a {0
 KRB5_FILE_FOUND.explanation=The configured file was successfully located.
 KRB5_FILE_FOUND.useraction=No action is required.
 
+# CWIML4517 already used
+
 IDMAP_INVALID_FORMAT=CWIML4518W: The {0} {1} value is malformed. The value must be a series of objectclass:attribute or *:attribute pairs, where each pair is separated by a semi-colon.
 IDMAP_INVALID_FORMAT.explanation=The userIdMap value must be in a valid format so that the objectclass and attribute can be parsed.
 IDMAP_INVALID_FORMAT.useraction=Update the userIdMap value so that it adheres to the specified format.
+
+# CWIML4519 and CWIML4520 already used
+
+KRB5_SERVICE_NOT_AVAILABLE=CWIML4521E: The {0} LdapRegistry attempted to bind to the Ldap server using Kerberos credentials for {1} principal name, but the KerberosService is not available. The bind authentication mechanism is {2}.
```

Comment:
suggested edit:
The {0} LdapRegistry attempted to bind to the Ldap server by using Kerberos credentials for the {1} principal name, but the KerberosService that provides these credentials is not available. The bind authentication mechanism is {2}.

If that's correct...

Reply:
Udpated